vue-components/tests/unit/build/postcssWikitScope.spec.ts

```diff
@@ -0,0 +1,36 @@
+const postcss = require( 'postcss' );
+const wikitScope = require( '../../../build/postcssWikitScope' );
+
+async function processWithPlugin( input: string ): Promise<{ css: string }> {
+	return postcss( [ wikitScope ] ).process( input, { from: undefined } );
+}
+
+describe( 'postcssWikitScope', () => {
+
+	it.each( [
+		[
+			'.foo {}',
+			'.wikit .foo, .foo.wikit {}',
+		],
+		[
+			'::before {}',
+			'.wikit ::before, .wikit::before {}',
+		],
+		[
+			'[type="button"] {}',
+			'.wikit [type="button"], .wikit[type="button"] {}',
+		],
+		[
+			'div [type=button] {}',
+			'.wikit div [type=button], div .wikit[type=button] {}',
```

Comment:
```suggestion
			'.wikit div [type=button], div.wikit [type=button] {}',
```

But let's discuss this first, maybe I'm confused.

Isn't it the case that we are expecting (by virtue of #116) the wikit class to be placed on the "root" node of the DOM?

So for the second selector (after the comma) we would not be looking for "wikit nodes which are a button residing inside of some div" (`div .wikit[type=button]`), but instead "button nodes residing in a div which is wikit" (`div.wikit [type=button]`).

Can this be generalized that for all "deep" selectors (with a space), only the part before the first space needs to be either scoped (`.wikit something`) or being the scope itself (`something.wikit`)?

Reply:
Nope, turns out I was confused. Adjusted the tests, and changed the code. Thanks!